packages/core/src/login/webview/vue/types.ts

```diff
@@ -59,6 +59,7 @@ export type AuthUiClick =
     | 'auth_toolkitCloseButton'
     | 'auth_reauthenticate'
     | 'auth_signout'
+    | 'auth_helpLinkClick'
```

Comment:
these are element ids. "click" happens to be the event they are added to, but mentioning "click" in the name is redundant (and confusing, if the element ids are used in other events).
```suggestion
    | 'auth_helpLink'
```